changelogs/fragments/57993-openssh_keypair-add-public-key-and-key-comment-validation.yml

```
@@ -0,0 +1,2 @@
+bugfix:
```

```suggestion
bugfixes:
```
Otherwise the tool won't accept this.